node/test/unit/tessel.js

```diff
@@ -1261,14 +1277,14 @@ exports['Tessel.Pin'] = {
       this.a.pin[pinIndex].on('rise', spy);
       this.b.pin[pinIndex].on('rise', spy);
 
-      test.equal(this.a.pin[pinIndex].interruptMode, 'rise');
-      test.equal(this.b.pin[pinIndex].interruptMode, 'rise');
+      test.equal(this.a.pin[pinIndex].interruptMode, 'change');
+      test.equal(this.b.pin[pinIndex].interruptMode, 'change');
 
       // Simulate receipt of pin state changes
-      this.a.sock.read.returns(new Buffer([REPLY.ASYNC_PIN_CHANGE_N + pinIndex]));
+      this.a.sock.read.returns(new Buffer([(REPLY.ASYNC_PIN_CHANGE_N + pinIndex) | (1 << 3)]));
       this.a.sock.emit('readable');
 
-      this.b.sock.read.returns(new Buffer([REPLY.ASYNC_PIN_CHANGE_N + pinIndex]));
+      this.b.sock.read.returns(new Buffer([(REPLY.ASYNC_PIN_CHANGE_N + pinIndex) | (1 << 3)]));
       this.b.sock.emit('readable');
```

Comment:
I just took a quick look at the node code for each of the modules I have (accel-mma84, relay-mono, sdcard, climate-si7020) and the only one that has references to `rise`, `fall` or `change` is the sdcard and it is only looking for the emitted value.

I can plug these in and test them using the examples, but it probably won't happen until next week.